src/hardware/vga_draw.cpp

```diff
@@ -351,42 +351,45 @@ static uint8_t * VGA_Draw_Linear_Line(Bitu vidstart, Bitu /*line*/) {
 	return ret;
 }
 
-static uint8_t* draw_linear_line_from_dac_palette(Bitu vidstart, Bitu)
+static uint8_t* draw_linear_line_from_dac_palette(Bitu vidstart, Bitu /*line*/)
 {
-	// Quick referencess
-	constexpr auto& max_pos        = vga.draw.linear_mask;
-	constexpr auto& line_bytes     = vga.draw.line_length;
-	constexpr auto palette_map     = vga.dac.palette_map;
-	constexpr auto bytes_per_pixel = sizeof(palette_map[0]);
-	assert(vga.draw.bpp / 8 == bytes_per_pixel);
-
-	// The running write position for the palettized VGA pixels
-	auto target_addr = TempLine;
-	const auto target_end = target_addr + line_bytes;
-
-	// Ensure the size of the palettized VGA line will fit
-	assert(templine_buffer.size() >= line_bytes);
-
-	const auto copy_palette = [&](auto pos, auto len) {
-		auto line_addr      = vga.draw.linear_base + pos;
-		const auto line_end = line_addr + len;
-		while (target_addr < target_end && line_addr < line_end) {
-			memcpy(target_addr, palette_map + *line_addr++, bytes_per_pixel);
-			target_addr += bytes_per_pixel;
+	const auto offset = vidstart & vga.draw.linear_mask;
+	const uint8_t *ret = &vga.draw.linear_base[offset];
+	const auto palette_map = vga.dac.palette_map;
```

Comment:
very minor: clang reformat

Reply:
(yup - that's the reverted content), the next commit fixes it up.